tests/test_posts.py

```diff
@@ -0,0 +1,58 @@
+from fastapi.testclient import TestClient
+from app.main import app
+from app.mongo_db import users_collection
+
+client = TestClient(app)
+
+
+# help functions
+def get_auth_header():
```

Comment:
This approach is good enough, but - really good tests shouldn't have any side effects. For example, you create the instance of the post before the test and know everything in advance, so you do not need to retrieve it for the tests dynamically.